src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -150,20 +84,28 @@ See more about [`SELECT` clause](https://docs.greptime.com/reference/sql/select)
 The `WHERE` clause can be used to filter data. It supports comparisons against string, boolean, and numeric values. The following SQL statement selects the rows where the `usage_user` is greater than 50.
 
 ```sql
-SELECT * FROM cpu_metrics WHERE usage_user > 50;
+SELECT * FROM cpu_metrics WHERE usage_user > 50 LIMIT 100;
+```
+
+You can also perform basic arithmetic in the `WHERE` condition.
+For example, the following query returns data where the sum of the field value `usage_user` and 10 is greater than 50.
+
+```sql
+SELECT * FROM cpu_metrics WHERE usage_user + 10 > 50 LIMIT 100;
```

Comment:
We may want a better example for this because `usage_user+10>50` does not make much sense.

Reply:
SELECT * FROM cpu_metrics WHERE usage_user + usage_system > 50 LIMIT 100;